detectors/node/opentelemetry-resource-detector-aws/package.json

```diff
@@ -41,26 +41,26 @@
     "access": "public"
   },
   "devDependencies": {
-    "@opentelemetry/api": "^1.0.0",
+    "@opentelemetry/api": "^1.9.0",
     "@opentelemetry/contrib-test-utils": "^0.45.0",
     "@opentelemetry/instrumentation-fs": "^0.19.0",
     "@opentelemetry/instrumentation-http": "^0.57.0",
-    "@types/mocha": "8.2.3",
-    "@types/node": "18.18.14",
-    "@types/sinon": "10.0.20",
-    "nock": "13.3.3",
-    "nyc": "15.1.0",
-    "rimraf": "5.0.10",
-    "sinon": "15.2.0",
-    "typescript": "4.4.4"
+    "@types/mocha": "10.0.10",
+    "@types/node": "22.10.6",
+    "@types/sinon": "17.0.3",
+    "nock": "13.5.6",
+    "nyc": "17.1.0",
+    "rimraf": "6.0.1",
+    "sinon": "19.0.2",
+    "typescript": "5.7.3"
```

Comment:
Are these changes needed? Looks like your editor may have done a dep upgrade en masse. Our tooling upgrades are usually done for teh whole repo one at a time.